google_pubsub_adapter/publisher/google_pubsub_publisher.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.pubsub_v1 import PublisherClient
+from django.conf import settings
+from rest_framework.renderers import JSONRenderer
+from herbie_core.services.message_publisher.abstract_publisher import AbstractPublisher
+from herbie_core.models.message_models_and_serializers import Message
+
+
+class GooglePubsubPublisher(AbstractPublisher):
+
+    def __init__(self):
+        self._logger = logging.getLogger(__name__)
+        self._publisher = PublisherClient()
+
+    def get_name(self) -> str:
+        return 'google_pubsub'
+
+    def send_message(self, message: Message):
+        serializer = message.get_serializer()
```

Comment:
I know the logic doesn't come from you, but when I read the code now it feels a bit strange to have the serializer as part of the message. But maybe we can leave it for now and brainstorm later if there is a better solution.

Reply:
Let's brainstorm about it later :+1: